Greenshot/releases/innosetup/setup.iss

```diff
@@ -214,42 +214,42 @@ Name: startup; Description: {cm:startup}
 
 [CustomMessages]
 ;Language names in the original language
-dexfranconia=Frängisch (Deutsch)
-arSY=العربية
-caCA=Català
-csCZ=Ceština
-daDK=Dansk
-elGR=ελληνικά
-esES=Español
-etEE=Eesti
-faIR=پارسی
-fiFI=Suomi
-frFR=Français
-frQC=Français - Québec
-heIL=עִבְרִית
-huHU=Magyar
-idID=Bahasa Indonesia
-itIT=Italiano
-jaJP=日本語
-kabDZ=Taqbaylit
-koKR=한국어
-ltLT=Lietuvių
-lvLV=Latviski
-nnNO=Nynorsk
-plPL=Polski
-ptBR=Português do Brasil
-ptPT=Português de Portugal
-roRO=Română
-ruRU=Pусский
-skSK=Slovenčina
-slSI=Slovenščina
-srRS=Српски
-svSE=Svenska
-trTR=Türk
-ukUA=Українська
-viVN=Việt
-zhCN=简体中文
-zhTW=繁體中文
+en.dexfranconia=Frängisch (Deutsch)
+en.arSY=العربية
+en.caCA=Català
+en.csCZ=Ceština
+en.daDK=Dansk
+en.elGR=ελληνικά
+en.esES=Español
+en.etEE=Eesti
+en.faIR=پارسی
+en.fiFI=Suomi
+en.frFR=Français
+en.frQC=Français - Québec
+en.heIL=עִבְרִית
+en.huHU=Magyar
+en.idID=Bahasa Indonesia
+en.itIT=Italiano
+en.jaJP=日本語
+en.kabDZ=Taqbaylit
+en.koKR=한국어
+en.ltLT=Lietuvių
+en.lvLV=Latviski
+en.nnNO=Nynorsk
+en.plPL=Polski
+en.ptBR=Português do Brasil
+en.ptPT=Português de Portugal
+en.roRO=Română
+en.ruRU=Pусский
+en.skSK=Slovenčina
+en.slSI=Slovenščina
+en.srRS=Српски
+en.svSE=Svenska
+en.trTR=Türk
+en.ukUA=Українська
+en.viVN=Việt
+en.zhCN=简体中文
+en.zhTW=繁體中文
```

Comment:
These translations have nothing to do with the english language, this is why it's not prefixed.
All names of the languages are how they are written in their own language!
I will not apply this part of the change.

Reply:
@Lakritzator 

I'm, so sorry but for me is not true.
For the rules of Inno Setup these strings are specific for English (master/base) language.

Please take care that a standard in "Inno Setup" to use "en." to higlight that are strings used when you select "English language" in the installer.
Than for mutuation these strings are used when is not available strings for other language (as it happen in this case for Italian language that has translation for it).

For who doesn't known well Inno Setup if you leave without "en." seems that these strings about language name cannot be translated. And is not true.

If you use "en.xxx" for other strings in .iss script you should use "en.xxx" also for language name.

Please use a coerent schematic approach at setup iss translation strings.


Thanks.

Reply:
@bovirus thanks a lot for your contributons :smiley: 

I understand your point. These strings _could_ be translated. IMO it does not add much value, though. It is a wide-spread pattern to list each language in their own name, and I think that most users are absolutely fine with it.

But then, why not do it anyway? Because of maintenance. Maintaining translations is a piece of extra-work for us. (Note that not every translator is able/willing to prepare pull requests as carefully as you do.) In theory, adding a new language would eventually result in multiple small pull requests to have its name translated to every other language. In practice, translating a list of language names is a tedious task for translators and keeping them up to date is awful, so we would most likely end up with a semi-translated language lists in most languages.

So I agree with @Lakritzator and would rather rather prefer to stick with the current pattern for language names.

And yeah... we definitely need to do something about our translation process, I know :confused:

Reply:
I haeva different vision about translation.

I think that a translation should translate as much as possible teh strinmgs and not leave if you can strings untranslated.

About extra work you can do one time and it was done.

Please remind  that I request to select by default all languages for installation.

Reply:
@Lakritzator 
@jklingen 

Please conisder to modify the instalelr script to remove 

- Manual selection of GUI languages
- Manual selection of plugins

and modify the installer script to

- Install all languages available
- Install al plugins available

In this way you 

- make simple the installer and relative maintenance
- are you sure that all languages/plugins are installed (not required additional setup to add others).

Thanks.